wgpu/examples/framework.rs

```diff
@@ -269,11 +269,7 @@ fn start<E: Example>(
     let spawner = Spawner::new();
     let mut config = wgpu::SurfaceConfiguration {
         usage: wgpu::TextureUsages::RENDER_ATTACHMENT,
-        format: *surface
```

Comment:
I'm not sold on needing a separate api here - especially if we remove the outer option.